keel-web/src/test/resources/examples/titus-cluster-with-artifact-example.yml

```diff
@@ -1,30 +1,33 @@
 ---
-kind: titus/cluster@v1
-metadata:
-  serviceAccount: my-email@spinnaker.io
-spec:
-  moniker:
-    app: keeldemo
-    stack: examples
-    detail: ec2v1
-  container:
-    organization: emburns
-    image: spin-titus-demo
-    tagVersionStrategy: semver-tag
-  locations:
-    account: titustestvpc
-    regions:
-      - name: us-west-2
-      - name: us-east-1
-  capacity:
-    min: 1
-    max: 1
-    desired: 1
-  overrides: {}
-  dependencies:
-    loadBalancerNames: []
-    securityGroupNames:
-      - keeldemo
-      - nf-datacenter
-      - nf-infrastructure
-    targetGroups: []
+application: fnord
+serviceAccount: delivery-engineering@netflix.com
+environments:
```

Comment:
where is the artifact here? :)

Reply:
Good point. These were previously just resources rather than full delivery configs, so I'm not sure why the file was called that originally.